distribution/lib/Standard/Test/0.0.0-dev/src/Faker.enso

```diff
@@ -1,6 +1,6 @@
 from Standard.Base import all
 
-polyglot java import java.util.Random
+polyglot java import java.util.Random as Java_Random
```

Comment:
Perfectly OK, in my opinion.

Reply:
In this module, `Random` used to point both to `Standard.Base.Random` module and `java.util.Random` polyglot type. So it was not clear which method should, e.g., `Random.new` call.